src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -32,20 +33,44 @@ interface DataAndTarget {
   target: S3Object;
 }
 
+interface deleteS3ObjectByKeys {
```

Comment:
`Objects`

same on the interface below.

Reply:
You can also leave out the `ByKeys` as it's implicit in what an S3 object is. (Also it's not just the keys, but a key and a version)

Reply:
done named as DeleteS3Objects